module/accelerometer/include/accelerometer.hpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2023 Robarm
+// All rights reserved
+#ifndef MODULE_ACCELEROMETER_INCLUDE_ACCELEROMETER_HPP_
```

Comment:
Separate copyright from include guard.

```suggestion
// All rights reserved

#ifndef MODULE_ACCELEROMETER_INCLUDE_ACCELEROMETER_HPP_
```